packages/web-components/fast-foundation/src/design-system/design-system.ts

```diff
@@ -95,11 +101,7 @@ export interface DesignSystemRegistrationContext {
      * @param callback - A callback to invoke if definition will happen.
      * @public
      */
-    tryDefineElement(
-        name: string,
-        type: Constructable,
-        callback: ElementDefinitionCallback
-    );
+    tryDefineElement(ctx: ElementDefinitionParams);
```

Comment:
Though not used directly by consumers (as far as I know), this would technically be a breaking change. Do we feel like we can get away with that on a minor release? /cc @chrisdholt

Reply:
Note: This is a better design as it would be more open to extension in the future without breaks. Just want to make sure that we're ok doing this on a minor release. I seriously doubt it will affect anyone, but again, technical breaking.